bitnami/airflow/templates/NOTES.txt

```diff
@@ -124,3 +124,4 @@ To connect to Airflow from outside the cluster, perform the following steps:
 
 {{ include "airflow.validateValues" . }}
 {{ include "airflow.checkRollingTags" . }}
+{{- include "common.warnings.resources" (dict "sections" (list "git.clone" "git.sync" "metrics" "scheduler" "web" "worker") "context" $) }}
```

Comment:
The section created is huge, and it is repeating always the same message. I think we should change the helper in a next iteration to avoid things like this:
```console
NAME: airflow
LAST DEPLOYED: Wed Feb 14 09:12:18 2024
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
CHART NAME: airflow
CHART VERSION: 16.6.0
APP VERSION: 2.8.1

** Please be patient while the chart is being deployed **

Airflow can be accessed via port 8080 on the following DNS name from within your cluster:

    airflow.default.svc.cluster.local

To connect to Airflow from outside the cluster, perform the following steps:

1.  Create a port-forward to the service:

    kubectl port-forward --namespace default svc/airflow 8080:8080 &
    echo "Airflow URL: http://127.0.0.1:8080"

2. Open a browser and access Airflow using the obtained URL.

3. Get your Airflow login credentials by running:

    export AIRFLOW_PASSWORD=$(kubectl get secret --namespace "default" airflow -o jsonpath="{.data.airflow-password}" | base64 -d)
    echo User:     user
    echo Password: $AIRFLOW_PASSWORD





WARNING: Section git.clone does not have the "resources" value set. Using "git.clone.resourcesPreset" is not recommended for production. For production workloads, please set the "git.clone.resources" value adapted to your workload needs. 
+info https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/


WARNING: Section git.sync does not have the "resources" value set. Using "git.sync.resourcesPreset" is not recommended for production. For production workloads, please set the "git.sync.resources" value adapted to your workload needs. 
+info https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/



WARNING: Section scheduler does not have the "resources" value set. Using "scheduler.resourcesPreset" is not recommended for production. For production workloads, please set the "scheduler.resources" value adapted to your workload needs. 
+info https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/


WARNING: Section web does not have the "resources" value set. Using "web.resourcesPreset" is not recommended for production. For production workloads, please set the "web.resources" value adapted to your workload needs. 
+info https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/


WARNING: Section worker does not have the "resources" value set. Using "worker.resourcesPreset" is not recommended for production. For production workloads, please set the "worker.resources" value adapted to your workload needs. 
+info https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
```

Reply:
Fixed in https://github.com/bitnami/charts/pull/23425